tests/plugins/test_orchestrate_build.py

```diff
@@ -235,12 +238,11 @@ def test_orchestrate_build(tmpdir, caplog, config_kwargs, worker_build_image, lo
         'conf_section': 'worker_x86_64',
         'conf_file': tmpdir + '/osbs.conf',
     }
-    if worker_build_image:
-        expected_kwargs['build_image'] = worker_build_image
     # Update with config_kwargs last to ensure that, when set
     # always has precedence over worker_build_image param.
     if config_kwargs is not None:
         expected_kwargs.update(config_kwargs)
+    expected_kwargs['build_image'] = 'fedora:latest'
```

Comment:
Hardcoding this is a bad idead

Reply:
so should I create constant and replace every occurrence in the file with that?

Reply:
I'm not sure why this needs to be set at all - `config_kwargs` should update this. A case where `config_kwargs` is empty is not valid for this change anymore and should be removed

Reply:
at line 244 expected_kwargs are updated with config_kwargs, which might set build_image (from osbs which functionality should remain same as per story description), but this change will be using orchestrators build_image, which is mocked in mock_workflow, to this 'fedora:latest'

Reply:
Oh, I see, `fedora:latest` in `worker_build_image` has confused me.